src/allmydata/test/test_client.py

```diff
@@ -123,14 +120,14 @@ def test_comment(self):
 
         def write_config(s):
             config = ("[client]\n"
-                      "introducer.furl = %s\n" % s)
+                      "helper.furl = %s\n" % s)
```

Comment:
Just to avoid triggering the deprecation by using `introducer.furl`.  Since the test is about the interaction between `#` and a furl item, I just picked a random other furl item (maybe this is an example of somewhere that a Hypothesis test would make the specification clearer?)

Reply:
Ahhh, I see .. okay :)